kickstarts/partials/post/node.ks.erb

```diff
@@ -0,0 +1,5 @@
+# Get the NodeJS RPM repo setup
+curl --silent --location https://rpm.nodesource.com/setup_6.x | bash -
```

Comment:
I'm checking on that.  FWIW - If someone can compromise that bash script, I'm sure they can compromise the source code to Node or the RPM.  So, yes blindly running a bash script maybe not the best, but just my $0.02